pkg/reconciler/subscription/subscription_test.go

```diff
@@ -288,6 +288,93 @@ func TestAllCases(t *testing.T) {
 					MarkSubscriptionReady,
 				),
 			}},
+		}, {}, {
```

Comment:
Is it possible to add a test with both `Group` and `APIVersion` specified?

Reply:
Added